main.rb

```diff
@@ -1,14 +1,12 @@
 require_relative 'book'
 require_relative 'label'
 require_relative 'item'
-require_relative 'game' # Include the game.rb file
-require_relative 'book_app'
-require_relative 'music_app'
```

Comment:
Hi @SAZZAD1Q2 ,

Please pull request first to prevent git conflict

Please ensure the main.rb is the latest or fetch first or pull request first
Please ensure no linter error